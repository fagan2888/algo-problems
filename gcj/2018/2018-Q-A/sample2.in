11
1 CS
2 CS
1 SS
6 SCCSSC
2 CC
3 CSCSS
1 CCCCCCCCCCCCCCCCCCCCCCCCCCCCS
1 CCCCCCCCCCCCCCCCCCCCCCCCCCCCCS
17 CSCCSCSSSSCSCCSCSSSSCSCCSCSSSS
18 CSCCSCSSSSCSCCSCSSSSCSCCSCSSSS
19 CSCCSCSSSSCSCCSCSSSSCSCCSCSSSS
