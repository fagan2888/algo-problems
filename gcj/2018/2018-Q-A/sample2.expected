Case #1: 1
Case #2: 0
Case #3: IMPOSSIBLE
Case #4: 2
Case #5: 0
Case #6: 5
Case #7: 28
Case #8: 29
Case #9: IMPOSSIBLE
Case #10: 132
Case #11: 131
